pkg/web/scene/scene.go

```diff
@@ -265,4 +273,8 @@ func WithConf(conf *config.Config) {
 	// Compute the sunriseEnabled boolean
 	enabled := conf.Sunrise.Enabled && conf.Email.Available()
 	sunriseEnabled = &enabled
+
+	// Compute the daybreakEnabled boolean
+	enabled = conf.Web.Daybreak.Enabled
```

Comment:
Based on your comment above, should this be as follows?

```suggestion
	enabled = conf.Web.Daybreak.Enabled && *sunriseEnabled
```

Reply:
We had discussed leaving it, in case a customer that didn't want to use Sunrise still wanted to see the counterparties in Daybreak for their internal compliance use, so I left it without the && part